ui/components/multichain/account-picker/account-picker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ButtonBase,
+  ICON_NAMES,
+  AvatarAccount,
+  Icon,
+  Text,
+} from '../../component-library';
+import {
+  Color,
+  FONT_WEIGHT,
+  Size,
+} from '../../../helpers/constants/design-system';
+
+export const AccountPicker = ({ account, onClick }) => {
+  return (
+    <ButtonBase
+      className="multichain-account-picker"
+      onClick={onClick}
+      backgroundColor={Color.transparent}
+    >
+      <AvatarAccount address={account.address} size={Size.SM} />
+      <Text
+        marginLeft={1}
+        marginRight={1}
+        fontWeight={FONT_WEIGHT.BOLD}
+        ellipsis
+      >
+        {account.name}
+      </Text>
+      <Icon name={ICON_NAMES.ARROW_DOWN} size={Size.SM} />
+    </ButtonBase>
+  );
+};
+
+AccountPicker.propTypes = {
+  account: PropTypes.object.isRequired,
+  onClick: PropTypes.func.isRequired,
```

Comment:
PropType descriptions help and will now show up next to the controls in storybook

Reply:
Done here: https://github.com/MetaMask/metamask-extension/pull/18177/commits/84a8998ce976eacf6c7aee4771ce9b3bf05b44da